Scripts/Embrace.cs

```diff
@@ -96,12 +100,21 @@ private void NoNullsError()
             Debug.LogError("null is not allowed through the Embrace SDK.");
         }
 
+        /// <summary>
+        /// Starts instrumentation of the Android application using the Embrace SDK. This should be called during creation of the application, as early as possible.
```

Comment:
Should be generic, this layer isn't specific to android.

Reply:
removed "android"